components/content/inspira/ui/aurora-background/AuroraBackground.vue

```diff
@@ -0,0 +1,58 @@
+<script setup lang="ts">
+import { cn } from "@/lib/utils";
+
+interface AuroraBackgroundProps {
+  radialGradient?: boolean;
+  class?: string;
+}
+
+const props = withDefaults(defineProps<AuroraBackgroundProps>(), {
+  radialGradient: true,
+});
+</script>
+
+<template>
```

Comment:
The format of the component file should be:

<template></template>
<script></script>
<style></style>

Reply:
Done